src/commands/force/user/create.ts

```diff
@@ -152,7 +155,7 @@ export class UserCreateCommand extends SfdxCommand {
     }
 
     // Set the alias if specified
-    if (this.flags.setalias) {
+    if (this.flags.setalias && typeof this.flags.setalias === 'string') {
```

Comment:
This is basically what `ensureString()` does in the ts-types lib.

Reply:
I don't want an error, just a check.  Did you mean `isString()`?